deploy-board/deploy_board/templates/configs/new_capacity_adv.html

```diff
@@ -193,7 +193,7 @@ <h4 class="panel-title pull-left">Capacity</h4>
                 }
             }),
         // TODO: The default host type comes from settings and should probably not be an abstract_name since they are not guaranteed to be unique.
-        selectedHostTypeValue: capacityCreationInfo.hostTypes.find(hostType => hostType.abstract_name === capacityCreationInfo.defaultHostType).id,
+        selectedHostTypeValue: capacityCreationInfo.hostTypes.find(hostType => hostType.abstract_name === capacityCreationInfo.defaultHostType) != undefined ? capacityCreationInfo.hostTypes.find(hostType => hostType.abstract_name === capacityCreationInfo.defaultHostType).id : "",
```

Comment:
Is it ok for this to be empty?

Reply:
the id is string. ideally this situation should never happen as it wasnt even being handled earlier. Let me know what other suggestion you have.

Reply:
alright, might be good to add a log line to detect this, so we can determine any exceptional cases. not sure the importance of this